RNTester/RNTesterUnitTests/RCTBundleURLProviderTests.m

```diff
@@ -20,12 +20,24 @@
 
 static NSURL *localhostBundleURL()
 {
-  return [NSURL URLWithString:[NSString stringWithFormat:@"http://localhost:8081/%@.bundle?platform=%@&dev=true&minify=false&app=com.apple.dt.xctest.tool", testFile, kRCTPlatformName]]; // TODO(macOS ISS#3536887)
+  return [NSURL
+      URLWithString:
+          [NSString
+              stringWithFormat:
+                  @"http://localhost:8081/%@.bundle?platform=%@&dev=true&minify=false&modulesOnly=false&runMdoule=true&app=com.apple.dt.xctest.tool",
```

Comment:
So you're purposefully changing the local host bundle url?

Reply:
Specifically, we're adding `modulesOnly` and `runMdoule` [sic].

The latter is a typo that was in the upstream repo that was fixed with b4c13929e2eeb2a697978b0d55f10139272326e3, but this is a few days after the cutoff for this particular PR.